cfme/infrastructure/provider/__init__.py

```diff
@@ -190,8 +190,11 @@ def hosts(self):
     def get_clusters(self):
         """returns the list of clusters belonging to the provider"""
         view = navigate_to(self, 'Clusters')
+        # TODO quadicon with JS vs non-JS doesn't have consistent attributes, no name for non-JS
```

Comment:
**OPTIONAL** expand the comment to make it more clear that the workaround is a temporary switch of the view